### PR TITLE
(doc) - mention include tag

### PR DESCRIPTION
**Pull request by jean-philippe-martin**
Following this checklist to help us incorporate your 
contribution quickly and easily:

 - (doc)
 - [x] Each commit in the pull request should have a meaningful subject line and body.
 - [x] Format the pull request title like `[MSHADE-XXX] - Fixes bug in ApproximateQuantiles`,
       where you replace `MSHADE-XXX` with the appropriate JIRA issue. Best practice
       is to use the JIRA issue title in the pull request title and in the first line of the 
       commit message.
 - [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [x] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will 
       be performed on your pull request automatically.
 - [x] You have run the integration tests successfully (`mvn -Prun-its clean verify`).

If your pull request is about ~20 lines of code you don't need to sign an
[Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf) if you are unsure
please ask on the developers list.

To make clear that you license your contribution under 
the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)
you have to acknowledge this by using the following check-box.

 - [x] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

 - [ ] In any other case, please file an [Apache Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

I didn't know about the `<include>` tag, and hadn't found it in the documentation. This PR makes it easier to find, putting it on the example page for the `<relocation>` tag.

### CODE REVIEW

File: src/site/apt/examples/class-relocation.apt.vm [hunk 1]
```diff
@@ -70,3 +70,20 @@ Relocating Classes
   into the package <<<org.shaded.plexus.util>>> by moving the corresponding JAR file entries and rewritting the
   affected bytecode. The class <<<Xpp3Dom>>> and some others will remain in their original package.
 
+
+  It's also possible to narrow the pattern with the "include" tag:
```
Comment (khmarbaise) [inline src/site/apt/examples/class-relocation.apt.vm]: Could you please replace `"` with `<<<` and the closing with `>>>` accordingly.

Comment (jean-philippe-martin) [reply src/site/apt/examples/class-relocation.apt.vm]: Sure thing! Done.

**Comment by khmarbaise** [comment]
Could you do another favour and squash those two commits into a single one....would greatly help me...afterwards I appreciate to merge this...

**Comment by jean-philippe-martin** [comment]
Sure thing @khmarbaise , done!

**Comment by khmarbaise** [comment]
Thanks for your contribution. This has been merged into master. Result can be seen here: https://gitbox.apache.org/repos/asf?p=maven-shade-plugin.git;a=commitdiff;h=3b68fe4796ad03301d3d21a4640fbc52b3b36123

**Comment by jean-philippe-martin** [comment]
Great, thank you!

**Pull request closed without merging**